jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -456,6 +457,62 @@ public void openProcessInstanceView(final String processInstanceId) {
         return task -> task.getProcessInstanceId() != null;
     }
 
+    public void bulkClaim(List<TaskSummary> taskSummaries) {
+        if (taskSummaries == null || taskSummaries.isEmpty()) {
+            return;
+        }
+        taskSummaries.parallelStream().forEach(taskSummary -> {
```

Comment:
Although using parallelStream could be more efficient, I would suggest using a plain _forEach_ to keep the behaviour more consistent.

Reply:
sure